src/main/java/org/wso2/carbon/connector/createMail.java

```diff
@@ -40,6 +40,4 @@ public void connect(MessageContext messageContext) throws ConnectException {
             throw new ConnectException(e);
         }
     }
-
```

Comment:
file name changed as CreateMail in https://github.com/wso2-extensions/esb-connector-gmail/pull/15